2018-2019/swampctf_2019/forensics/neo_50/README.md

```diff
@@ -0,0 +1,15 @@
+# Forensics: Neo
+## SwampCTF
+
+This was a 50 point challenge in the forensics category from SwampCTF. This jpeg file is provided for the challenge:
+<p align="center">
+  <img src=red_pill.jpeg>
+</p>
+
+This steganography challenge is actually very simple. All that was needed was two commands:
```

Comment:
Where did u find `commands.PNG`?

Reply:
It's just a screenshot I took of what I did. The file should be included in the folder: 2018-2019/swampctf_2019/forensics/neo_50/commands.PNG